xml/security_ldap_replication.xml

```diff
@@ -0,0 +1,469 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook" 
+       xmlns:xi="http://www.w3.org/2001/XInclude" 
+       xmlns:xlink="http://www.w3.org/1999/xlink" 
+       version="5.0" 
+       xml:id="sec-security-ldap-replication">
+ <title>Setting up replication</title>
+
+ <para>
+  &ds389; supports replicating its database content between multiple servers.
+  According to the type of replication, this provides:
+ </para>
+
+ <itemizedlist>
+  <listitem>
+   <para>
+    Faster performance and response times
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Fault tolerance and failover
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Load balancing
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    High availability
+   </para>
+  </listitem>
+ </itemizedlist>
+
+ <sect2 xml:id="sec-security-ldap-replication-async">
+  <title>Asynchronous writes</title>
+  <para>
+   &ds389a; manages replication differently than other databases. Replication
+   is asynchronous, and eventually consistent. This means:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Any write or change to a single server is immediately accepted.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     There is a delay between a write and replication to other servers.
```

Comment:
A delay between a write finishing on one server, and then replicating and being visible on other servers. Might be a bit clearer to indicate that the other servers "lag".